.pylintrc

```diff
@@ -23,7 +23,20 @@ disable=
  old-style-class,
 # "R" Refactor recommendations
  too-few-public-methods,
+# we need, some time, to disable pylint locally :)
+ locally-disabled,
```

Comment:
What's the purpose of this one?

Reply:
On py2.7, if you disable pylint in particular line, pylint raise a locally-disabled warning these config makes pylint happy and doesn't show these warning when you disable inline.

Reply:
Ah, ok. Then please update the comment, it's quite cryptic/misleading. Change it to something like `# do not warn about pylint checks being disabled through comments in the code`.

Reply:
Done!